src/Command/PurgeCommand.php

```diff
@@ -0,0 +1,359 @@
+     * @return int
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    private function purgeLeads(array $leads, array $masterForm)
+    {
+        $deleted = 0;
+        $ids = implode(',', array_keys($leads));
+
+        if (!empty($ids)) {
+            $deleted = $this->db->executeUpdate(
+                "DELETE FROM tl_lead WHERE id IN(".$ids.")"
+            );
+
+            $logLevel = LogLevel::INFO;
+            $logMessage = 'Purged '.(int)$deleted.' leads for master form "'.$masterForm['title'].'"';
+            $this->logger->log($logLevel, $logMessage, array('contao' => new ContaoContext(__METHOD__, $logLevel)));
+        }
+
+        return $deleted;
+    }
+
+    /**
+     * @param array $leads
+     * @return array
+     */
+    private function getAllLeadsData(array $leads)
+    {
+        $leadsData = [];
+
+        if (!empty($leads)) {
+
+            $ids = implode(',', array_keys($leads));
+
+            $rows = $this->db->fetchAll(
+                "SELECT d.*, f.type AS field_type FROM tl_lead_data d
+                      LEFT JOIN tl_form_field f ON d.field_id = f.id
+                      WHERE d.pid IN(".$ids.")"
+            );
+
+            foreach ($rows as $row) {
+                $leadsData[$row['id']] = $row;
+            }
+        }
+
+
+        return $leadsData;
+    }
+
+    /**
+     * @param array $leadsData
+     * @param array $masterForm
+     * @return int
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    private function purgeLeadsData(array $leadsData, array $masterForm)
+    {
+        $deleted = 0;
+
+        if (!empty($leadsData)) {
+
+            $ids = implode(',', array_keys($leadsData));
+
+            $deleted = $this->db->executeUpdate(
+                "DELETE FROM tl_lead_data WHERE id IN(".$ids.")"
+            );
+
+            $logLevel = LogLevel::INFO;
+            $logMessage = 'Purged '.(int)$deleted.' leads data for master form "'.$masterForm['title'].'"';
+            $this->logger->log($logLevel, $logMessage, array('contao' => new ContaoContext(__METHOD__, $logLevel)));
+        }
+
+        return $deleted;
+    }
+
+    /**
+     * @param array $leadsData
+     * @param string $leadPurgeUploads
+     * @return array
+     */
+    private function getUploads(array $leadsData, string $leadPurgeUploads)
+    {
+        $uploads = [];
+        if (!empty($leadsData) && !empty($leadPurgeUploads)) {
+            foreach ($leadsData as $data) {
+                if ('upload' === $data['field_type']) {
+                    $uploads[$data['id']] = $this->getUploadFileModel($data['value']);
+                }
+            }
+        }
+
+        return $uploads;
+    }
+
+    /**
+     * @param string $value
+     * @return FilesModel|null
+     */
+    private function getUploadFileModel(string $value)
+    {
+        if (!Validator::isUuid($value)) {
+            return null;
+        }
+
+        $filesModel = FilesModel::findByUUid($value);
```

Comment:
If you want to test this class, use `$this->framework->getAdapter(FilesModel::class)->findByUuid($value)`.